api/graphql/schema.graphql

```diff
@@ -1872,6 +1877,13 @@ type Mutation {
   markNotificationAsUnread(id: UUID!): Notification @guard
   deleteNotification(id: UUID!): Notification @guard
   markAllNotificationsAsRead: [Notification] @guard
+  updateIgnoredNotifications(
+    id: UUID!
+    ignoredEmailNotifications: [NotificationFamily]
+      @rename(attribute: "ignored_email_notifications")
+    ignoredInAppNotifications: [NotificationFamily]
+      @rename(attribute: "ignored_in_app_notifications")
+  ): User @update @guard @validator
```

Comment:
It seems as though any user can update another users ignored notifications here. Should we remove the ID and make it so running this mutation always affects the authenticated user or at least add a permission like `update-own-ignoredNotifcations`? 

Dropping a mention to @tristan-orourke to see his thoughts on best approach.

Reply:
Agreed - I think the query to get notifications only works with the logged in user, right? The mutations to modify them should work the same.

Reply:
Updated with [53fd40f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9785/commits/53fd40f88865224c7800e4ee0e5720f954959ab5) to just update the requesting users data.